### ISSUE TITLE
Migrating profiles from chromium?

### ISSUE THREAD

**Issue by dm17** [Eloston/ungoogled-chromium#1166]
Is there a standard/tested way to migrate profiles from our chromium setup to ungoogled-chromium? I'm mostly concerned with bookmarks, extensions, sessions, and settings.
Thanks!

**Comment by PF4Public** [comment]
Shouldn't they be readily exchangeable with each other?

**Comment by dm17** [comment]
> Shouldn't they be readily exchangeable with each other?
> I mean, copy-paste the whole profile and be done with it.

Would be good to know!

**Comment by Eloston** [comment]
Yes, you should be able to just use the profiles directly in ungoogled-chromium. If you cannot, then it's a bug.

**Comment by hussain1406** [comment]
Hi, I tried installing ungoogled-chromium in Linux Mint 21.2 but the previously installed version of chromium (from Linux Mint Repo) was v122. And the latest version in the OBS repository which was being installed on my OS was 112, when i tried opening ungoogled-chromium after installing, it gave me the following error:
```
[49920:49945:0314/142904.493711:ERROR:login_database_async_helper.cc(54)] Could not create/open login database.
```

This is just a part of the error. I think it shows what the problem was. Now I don't have the complete 4 line error as I have installed chromium from the Linux Mint Repo again as the previous was not working.

My question is, is there a way to migrate the data between two different/incompatible versions of chromium? (in this case ungoogled-chromium and chromium)
